.cicd/pipeline.yml

```diff
@@ -1,18 +1,21 @@
 steps:
   - wait
 
-  - label: ":webhook: Trigger Travis CI Build"
-    command: ". ./.cicd/trigger-travis.sh"
-    agents:
-      queue: "automation-basic-builder-fleet"
-    timeout: ${TIMEOUT:-2}
 
   - trigger: "eos-vm-base-images-beta"
     label: ":docker: Docker - Ensure Base Images Exist"
     build:
       commit: "${BUILDKITE_COMMIT}"
       branch: "${BUILDKITE_BRANCH}"
-      
+
+  - wait
+
+  - label: ":webhook: Trigger Travis CI Build"
```

Comment:
Nice catch, dude!!
This definitely has to be after the base images trigger.

Reply:
🍻